shaazk/Broadcaster
# #170038150 Add database scripts

#### What does this PR do?
- Adds DB scripts and data-structures
#### Description of Task to be completed?
- Add DB scripts
- Create tables in the database
- Create user and incidents
- Delete Incidents
- Update comment 
#### How should this be manually tested?
 - Clone Broadcaster on your computer and open it in your VS Code or any other  IDE. 
- Run npm run dev, go to your postman and eg. sign up on http://localhost:4000/api/v2/auth/signup but make sure that you have your Postgres server running. - In your postman, click send and check-in your Broadcaster_bd to checked the data of the new user
#### What are the relevant pivotal tracker stories?
- [#170038150](https://www.pivotaltracker.com/story/show/170038150)
